### ISSUE TITLE
Is mcrypt still required?

### ISSUE THREAD

**Issue by djmattyg007** [agendav/agendav#233]
The documentation and the ansible playbook all claim that mcrypt is still required, but I can't see it used at all in the code.

**Comment by pixelbrackets** [comment]
Yep, cant find any code which makes use of mcrypt. This may be removed from all docs (readme and installation guide). PRs welcome.

**Comment by pixelbrackets** [comment]
Removed
